plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -141,13 +144,21 @@ private String getJenkinsWar() {
 
         File pluginFileLocation = getPluginTxt();
         if (Files.exists(pluginFileLocation.toPath())) {
-            try (Scanner scanner = new Scanner(pluginFileLocation, StandardCharsets.UTF_8.name())) {
+            try (InputStreamReader inputStreamReader = new InputStreamReader(new FileInputStream(pluginFileLocation),
+                    StandardCharsets.UTF_8); BufferedReader bufferedReader = new BufferedReader(inputStreamReader)) {
```

Comment:
You could probably condense this down to:
```suggestion
                    StandardCharsets.UTF_8); BufferedReader bufferedReader = BufferedReader br = Files.newBufferedReader(Paths.get(fileName))) {
```
bypassing the need for the input stream.